src/main/java/de/unistuttgart/bugfinder/gameresult/OverworldResultDTO.java

```diff
@@ -17,7 +17,7 @@
 @FieldDefaults(level = AccessLevel.PRIVATE)
 public class OverworldResultDTO {
 
-  final String game = "BUGFINDER";
+  static final String game = "BUGFINDER";
```

Comment:
```suggestion
  final String game = "BUGFINDER";
```
This can not be static. The `game` attribute is part of the OverworldResultDTO and the overworld backend is expecting this in the request object as an attribute.